scss/components/_avatar.scss

```diff
@@ -31,6 +31,7 @@
   button.avatar-edit-overlay {
     align-items: center;
     background: rgba($black, .75);
+    border-radius: 50%;
```

Comment:
O.o why was this not already a thing.

Reply:
Since the parent had the border-radius and hidden overflow, the button probably didn't need it except in Safari.

Reply:
Yeah I normally bank on that hidden overflow since matching the border-radius can look ugly around the edges in certain circumstances. since we have a light border around the image, though, it should be fine.